packages/instrumenter/src/frameworks/angular-ignore.ts

```diff
@@ -0,0 +1,78 @@
+import type { NodePath as BabelNodePath } from '@babel/core';
+
+import { Ignorer, NodePath } from '@stryker-mutator/api/ignore';
+
+declare module '@stryker-mutator/api/ignore' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface NodePath extends BabelNodePath {}
+}
+
+const ANGULAR_SIGNAL_IO_FUNCTIONS = ['input', 'model', 'output'];
+
+export class AngularIgnore implements Ignorer {
+  public shouldIgnore(path: NodePath): string | undefined {
+    return this.isObjectInsideIOFunction(path)
+      ? 'Angular signal based input, model and output functions configuration object cannot be mutated as that causes issues with the ivy compiler.'
```

Comment:
```suggestion
      ? 'Angular compiler doesn't support mutations here.'
```

This is more to the point IMO